src/app/(proper_react)/(redesign)/(public)/LandingView.test.tsx

```diff
@@ -89,7 +89,10 @@ describe("When Premium is not available", () => {
         user: {
           email: "arbitrary@example.com",
         },
+        expires: "2023-06-18T14:48:00.000Z",
       },
+      status: "authenticated",
+      update: () => new Promise(() => null),
```

Comment:
I think this will result in a `Promise` that is `pending`, do you actually want to return a `Promise` that resolves to `null` here? I think that should be:
```
new Promise(resolve => resolve(null))
```

Or I think this is also equivalent for your case and shorter:
```
Promise.resolve(null)
```

Reply:
Thanks! For this test, the intention is to return a `Promise` that resolved to `null`. The type declaration for NextAuth’s [update method](https://next-auth.js.org/getting-started/client#updating-the-session) is `update(data?: any): Promise<Session | null>`. Updated to your suggestion in [e72832b](https://github.com/mozilla/blurts-server/pull/4696/commits/e72832bbed8f93db6479794e3c70f51e7d901ef9).